metricbeat/module/kafka/consumergroup/_meta/fields.yml

```diff
@@ -43,6 +43,10 @@
       type: keyword
       description: custom consumer meta data string
 
+    - name: consumer_lag
+      type: long
+      description: consumer lag for partition/topic
```

Comment:
probably worth it explaining what this is, as an important metric: the difference between the partition offset and consumer offset